src/components/Tool/Appeal/Flow/ContactFlowRow/ContactFlowRow.tsx

```diff
@@ -80,42 +102,45 @@ export const ContactFlowRow: React.FC<Props> = ({
       <DraggableBox
         style={{
           flexWrap: 'wrap',
+          paddingLeft: theme.spacing(1),
         }}
       >
-        <Box
-          style={{
-            display: 'flex',
-            justifyContent: 'space-between',
-            alignItems: 'center',
-            width: '100%',
-          }}
-        >
-          <Box display="flex" alignItems="center" width="100%">
-            <StyledAvatar src={avatar || ''} />
-            <Box display="flex" flexDirection="column" ml={2} draggable>
+        <FlexCenterAlignedBox justifyContent={'space-between'}>
+          <FlexCenterAlignedBox>
+            <StarContactIconButton
+              accountListId={accountListId}
+              contactId={id}
+              isStarred={starred || false}
+              size="small"
+            />
+            <Box display="flex" flexDirection="column" ml={1} draggable>
               <ContactLink onClick={() => onContactSelected(id, true, true)}>
                 {name}
               </ContactLink>
               <Typography variant="body2">
                 {getLocalizedContactStatus(t, contactStatus)}
               </Typography>
             </Box>
+          </FlexCenterAlignedBox>
+          <Box display="flex" width={'40px'}>
```

Comment:
With these style changes it is overflowing and a X scrollbar is showing up for each flow column. Could we remove that?

Reply:
Fixed.